scripts/pr_check.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+# This script is designed to ensure all relevant header and source files contain correct copyright
+
+set +e
+
+sudo apt-get install dos2unix
+
+ok=0
+fixed=0
+
+function check_folder {
+    for filename in $(find $1 -type f \( -iname \*.cpp -o -iname \*.h -o -iname \*.hpp -o -iname \*.js -o -iname \*.bat -o -iname \*.sh -o -iname \*.txt \)); do
+          if [[ $(grep -oP "Software License Agreement" $filename | wc -l) -ne 0 ]]; then
+               echo "[WARNING] $filename contains 3rd-party license agreement"
+          else
+               if [[ ! $filename == *"libuvc"* ]]; then
+                    # Only check files that are not .gitignore-d
+                    if [[ $(git check-ignore $filename | wc -l) -eq 0 ]]; then
+                         if [[ $(grep -oP "(?<=\(c\) )(.*)(?= Intel)" $filename | wc -l) -eq 0 ]]; then
+                              echo "[ERROR] $filename is missing the copyright notice"
+                              ok=$((ok+1))
+
+                              if [[ $2 == *"fix"* ]]; then
+                                   if [[ $(date +%Y) == "2019" ]]; then
```

Comment:
@dorodnic  I am sure that we will forget to update when we get to 2020 :)

Reply:
This is mostly due to my lack of bash knowledge. I'm using [`ex -sc '1i|// Copyright(c) 2019 Intel Corporation`](https://github.com/IntelRealSense/librealsense/pull/3265/files/dcc28304a441ad64fd49c18de8823bd73de52df1..6c19763c09a151f903cfd2f90e8e41efc042ff45#diff-a05f7d7d629be0edb2c306022e87156dR28) to append copyright before the first line.
However, if you try to use variables in that line (`ex -sc '1i|// Copyright(c) $(date +%Y) Intel Corporation`) you will just see "// Copyright(c) $(date +%Y) Intel Corporation" appended to the file. I tried to figure this out for half-an-hour and eventually gave up. 
If someone knows how to do this better, will be happy to change it.